pom.xml

```diff
@@ -242,7 +243,26 @@
 			</exclusions>
 		</dependency>
 
-		<dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
```

Comment:
We need to consider JUnit 5 support from a build-perspective. We need to evaluate what's the best approach, ideally, we can import JUnit 5 dependencies in the same way as we do it with Junit 4.

Reply:
that's right, but none of the parent spring-data poms provides a junit 5 version. I would have preferred to not define the dependency and the version here

Reply:
Can you file a ticket against https://github.com/spring-projects/spring-data-build/?

Reply:
You also do not want to manage the individual versions for JUnit 5 artifacts.

Rather, you should import the `junit-bom` for that.

And if you want to use JUnit Jupiter, you'll probably want to depend on the `junit-jupiter` aggregating artifact (similar to a Spring Boot starter POM).

Reply:
fixed that (thanks @mp911de for adding that to spring-data-build)

Reply:
Spring Data Build is pulling in [`junit-jupiter` and `junit-vintage-engine` dependencies](https://github.com/spring-projects/spring-data-build/blob/master/parent/pom.xml#L945-L955).